pkg/storage/unified/apistore/store.go

```diff
@@ -171,6 +171,15 @@ func (s *Storage) convertToObject(data []byte, obj runtime.Object) (runtime.Obje
 	return obj, err
 }
 
+// GetCurrentResourceVersion implements storage.Interface.
+// See: https://github.com/kubernetes/kubernetes/blob/v1.33.0/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go#L647
+func (s *Storage) GetCurrentResourceVersion(ctx context.Context) (uint64, error) {
+	// The RV is based on the current micro time
+	// Unlike etcd, each namespace will have its own max RV, so this seems like a reasonable compromise
+	now := time.Now().UnixMicro()
+	return uint64(now), nil
+}
```

Comment:
^^^ this is a new function, and I am not sure what the proper implementation should be!

@chaudyg any ideas

Reply:
previous implementation:
https://github.com/kubernetes/kubernetes/blob/v1.32.4/staging/src/k8s.io/apiserver/pkg/storage/util.go#L86

Reply:
I suspect we should use the max resource version from the resource version table 🤔

Reply:
> This method issues an empty list request and reads only the ResourceVersion from the object metadata.

That is what we previously did... but in v33 it was added without a resource context (eg... the etcd RV, not one scoped to a resource)

I moved this logic to https://github.com/grafana/grafana/pull/106077 since it is non-trivial and hard to review in context with all the codegen + import changes